docs/standard/parallel-programming/how-to-write-messages-to-and-read-messages-from-a-dataflow-block.md

```diff
@@ -1,51 +1,57 @@
 ---
-title: "How to: Write Messages to and Read Messages from a Dataflow Block"
-ms.date: "03/30/2017"
+title: "How to: Write and read messages from a Dataflow block"
+ms.date: 09/10/2020
 ms.technology: dotnet-standard
-dev_langs: 
+dev_langs:
   - "csharp"
   - "vb"
-helpviewer_keywords: 
+helpviewer_keywords:
   - "Task Parallel Library, dataflows"
   - "TPL dataflow library, reading and writing messages"
 ms.assetid: 1a9bf078-aa82-46eb-b95a-f87237f028c5
 ---
-# How to: Write Messages to and Read Messages from a Dataflow Block
-This document describes how to use the TPL Dataflow Library to write messages to and read messages from a dataflow block. The TPL Dataflow Library provides both synchronous and asynchronous methods for writing messages to and reading messages from a dataflow block. This document uses the <xref:System.Threading.Tasks.Dataflow.BufferBlock%601?displayProperty=nameWithType> class. The <xref:System.Threading.Tasks.Dataflow.BufferBlock%601> class buffers messages and behaves as both a message source and as a message target.  
+
+# How to: Write and read messages from a Dataflow block
+
+This document describes how to use the TPL Dataflow Library to write messages to and read messages from a dataflow block. The TPL Dataflow Library provides both synchronous and asynchronous methods for writing messages to and reading messages from a dataflow block. This document uses the <xref:System.Threading.Tasks.Dataflow.BufferBlock%601?displayProperty=nameWithType> class. The <xref:System.Threading.Tasks.Dataflow.BufferBlock%601> class buffers messages and behaves as both a message source and as a message target.
```

Comment:
Wording and style suggestions (document --> article, expand acronym on first use, etc.)

```suggestion
This article describes how to use the Task Parallel Library (TPL) Dataflow Library to write messages to and read messages from a dataflow block. The TPL Dataflow Library provides both synchronous and asynchronous methods for writing messages to and reading messages from a dataflow block. This article shows how to uses the <xref:System.Threading.Tasks.Dataflow.BufferBlock%601?displayProperty=nameWithType> class. The <xref:System.Threading.Tasks.Dataflow.BufferBlock%601> class buffers messages and behaves as both a message source and a message target.
```